docs/pages/identity-security/session-summaries.mdx

```diff
@@ -0,0 +1,339 @@
+
+If a match is found, Teleport proceeds to summarize the session recording. It
+uses an inference model configuration indicated by the policy to generate a
+summary using an external inference provider, such as OpenAI. There is no way
+to run this process on demand.
+
+There are three fundamental concepts to that underpin session summarization:
+
+- **Inference providers** do the heavy lifting of summarization. Currently,
+  Teleport only supports OpenAI or compatible inference providers.
+- **Inference models** indicate *how* sessions get summarized: they are
+  Teleport configuration resources that provide model settings.
+- **Inference policies** indicate *what* gets summarized and which inference
+  model is used in given case.
+
+## Prerequisites
+
+- A running Teleport Enterprise cluster v18.2.0 or later with Identity Security
+  enabled.
+- Access to OpenAI or OpenAI-compatible inference API, either public or
+  self-hosted.
+
+## Step 1/4: Configure access to inference configuration
+
+In order to manage the summary inference configuration, you need a write access
+to `inference_model`, `inference_secret`, and `inference_policy` resources,
+which is granted by default through the built-in `editor` role.
+
+You can either use the built-in roles or create a new one to make sure you have
+access to all necessary resources. To create a new role:
+
+1. Create a `summarizer-admin.yaml` file with the following content:
+
+   ```yaml
+   kind: role
+   metadata:
+     name: summarizer-admin
+   spec:
+     allow:
+       rules:
+       - resources:
+         - inference_model
+         - inference_secret
+         - inference_policy
+         verbs:
+         - read
+         - list
+         - create
+         - update
+         - delete
+   version: v7
+   ```
+
+1. Apply your changes:
+
+   ```code
+   $ tctl create -f summarizer-admin.yaml
+   ```
+
+   (!docs/pages/includes/create-role-using-web.mdx!)
+
+1. (!docs/pages/includes/add-role-to-user.mdx role="summarizer-admin"!)
+
+## Step 2/4: Create the inference configuration resources
+
+Let's start by defining an `inference_model` resource. It tells Teleport how to
+summarize a session by pointing it to a large language model (LLM) API. In
+particular, it contains an API-specific model name. Note that the *Teleport
+inference model name* and *API-specific model name* are two different things.
+This allows, for example, using the same OpenAI model with different
+temperature settings or using different API gateways by creating multiple
+Teleport inference models with the same OpenAI model name.
+
+To communicate with OpenAI, an inference model needs an API key. It's stored
+separately in an `inference_secret` resource and referenced by the
+`inference_model`'s `spec.openai.api_key_secret_ref` field.
+
+<Admonition type="warning">
+To protect your API key, once written, an `inference_secret` value cannot be
+retrieved through `tctl` or Teleport API. It can only be updated or deleted.
```

Comment:
Not even with `tctl get --with-secrets`?

What happens if you try to `tctl edit` the resource?